test/unit/app/plugins/router.test.js

```diff
@@ -26,14 +25,61 @@ describe('routes plugin test', () => {
       '/healthy',
       '/healthz',
       '/claim/cookies',
+      '/claim/endemics',
       '/claim/signin-oidc',
       '/claim/assets/{path*}',
-      '/claim/cookies'
+      '/claim/endemics/biosecurity',
+      '/claim/endemics/check-answers',
+      '/claim/endemics/confirmation',
+      '/claim/endemics/date-of-testing',
+      '/claim/endemics/date-of-visit',
+      '/claim/endemics/disease-status',
+      '/claim/endemics/number-of-fluid-oral-samples',
+      '/claim/endemics/number-of-samples-tested',
+      '/claim/endemics/number-of-species-tested',
+      '/claim/endemics/pi-hunt',
+      '/claim/endemics/pi-hunt-all-animals',
+      '/claim/endemics/pi-hunt-recommended',
+      '/claim/endemics/sheep-endemics-package',
+      '/claim/endemics/sheep-test-results',
+      '/claim/endemics/sheep-tests',
+      '/claim/endemics/species-numbers',
+      '/claim/endemics/test-results',
+      '/claim/endemics/test-urn',
+      '/claim/endemics/vaccination',
+      '/claim/endemics/vet-name',
+      '/claim/endemics/vet-rcvs',
+      '/claim/endemics/vet-visits-review-test-results',
+      '/claim/endemics/which-species',
+      '/claim/endemics/which-type-of-review',
+      '/claim/cookies',
+      '/claim/endemics/biosecurity',
+      '/claim/endemics/check-answers',
+      '/claim/endemics/date-of-testing',
+      '/claim/endemics/date-of-visit',
+      '/claim/endemics/disease-status',
+      '/claim/endemics/number-of-fluid-oral-samples',
+      '/claim/endemics/number-of-samples-tested',
+      '/claim/endemics/number-of-species-tested',
+      '/claim/endemics/pi-hunt',
+      '/claim/endemics/pi-hunt-all-animals',
+      '/claim/endemics/pi-hunt-recommended',
+      '/claim/endemics/sheep-endemics-package',
+      '/claim/endemics/sheep-test-results',
+      '/claim/endemics/sheep-tests',
+      '/claim/endemics/species-numbers',
+      '/claim/endemics/test-results',
+      '/claim/endemics/test-urn',
+      '/claim/endemics/vaccination',
+      '/claim/endemics/vet-name',
+      '/claim/endemics/vet-rcvs',
+      '/claim/endemics/vet-visits-review-test-results',
+      '/claim/endemics/which-species',
+      '/claim/endemics/which-type-of-review'
     ])
   })
 
   test('routes included - endemics enabled', async () => {
```

Comment:
do we need this test anymore? anything calling out 'endemics enabled' or 'endemics disabled' can be renamed, and remove the one where its disabled.